src/App.js

```diff
@@ -22,7 +22,7 @@ class App extends Component {
   };
 
   getFeed = e => {
-    this.setState({ fetching: !this.state.fetching });
+    this.setState(state => ({ ...state, fetching: !this.state.fetching }));
```

Comment:
Since we are accessing previous state here, would it be easier to read if we only update the part of state we need?

```suggestion
    this.setState(({ fetching }) => ({ fetching: !fetching }));
```

Reply:
🔝 !

Reply:
i think i like this approach best